### PR TITLE
Drop six

### PR DESCRIPTION
**Pull request by alyohea**
- Drop six from requirements
- Fix some minor code things

### CODE REVIEW

File: demo/tests/test_fields.py [hunk 1]
```diff
@@ -48,11 +46,6 @@ def test_unkown_string(self):
         with self.assertRaises(UnregisteredAgnocompleteException):
             AgnocompleteField('MEUUUUUUH')
 
-    def test_unicode(self):
```
Comment (alyohea) [inline demo/tests/test_fields.py:51]: Drop this test since string are in unicode in py3

Comment (alyohea) [inline demo/tests/test_fields.py:51]: Drop this test since strings are unicode in py3 by default

**Review by ewjoachim — APPROVED** [review]
Nice. You could run [pyupgrade](https://github.com/asottile/pyupgrade) to avoid doing all that by hand (and maybe even add pre-commit)

**Comment by alyohea** [comment]
I run the code with pyupgrade, now it should be cleaner

**Pull request merged**